ZenPacks/zenoss/Import4/service_definition/imp4opentsdb.json

```diff
@@ -93,6 +94,18 @@
          {
            "Name": "HBase Regionservers up",
            "Script": "{{with $rss := (child (child (parent .) \"HBase\") \"RegionServer\").Instances }}wget -q -O- http://localhost:61000/status/cluster | grep '{{$rss}} live servers'{{end}}"
+         },
+         {
+           "Name": "OpenTSDB up",
+           "Script": "/opt/opentsdb/build/tsdb uid --config=/opt/zenoss/etc/opentsdb/opentsdb.conf grep metrics jvm.uptime"
+         },
+         {
+           "Name": "DMD UUID obtained",
+           "Script": "[ -f /import4/dmd_uuid.txt ]"
```

Comment:
Yes, I believe that the dmd.uuid will go back to the old one  after the zodb is imported. Thus, this container can only run correctly after this piece of information is restored. Please advice if this is not the case.

Thanks!